### DISCUSSION TITLE
units and blades

### DISCUSSION THREAD

**Discussion by mindevis** [netbox-community/netbox#17630] [Q&A]
Hello everyone, please tell me, I have a 2-unit server inside which there are 4 blades, when I configure device bays they are displayed as Server (4/4)
But I need all these 4 servers to be displayed inside the unit, as child elements of Server(4/4), so that I could, for example, click on the child element Server(4/4) - ServerSlotA and get into its information, how is it possible to implement this? Such functionality is available in third-party software that has not been updated for a long time (racktables)

**Comment by PieterL75** [comment]
I assume you are talking about the rack views ? 
You can click on the 2-unit server, and then on the 'device bays' tab, but I guess that is not what you want to see..

There is a FR open, to make the popup customizalbe, so that this could include the devices in that chassis.. Feel free to upvote it if you think this can be of help
https://github.com/netbox-community/netbox/issues/14205#issuecomment-2257379257
